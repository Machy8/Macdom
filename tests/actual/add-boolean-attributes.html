input $text beer steak muhehe