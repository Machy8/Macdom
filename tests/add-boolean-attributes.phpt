<?php

use Tester\Assert;


require __DIR__ . '/bootstrap.php';

$macdom->setup->addBooleanAttributes = "beer steak muhehe";
$macdom->setup->compressCode = TRUE;

Assert::matchFile(
	EXPECTED . "/add-boolean-attributes.html",
	$macdom->compile(file_get_contents(ACTUAL . "/add-boolean-attributes.html"))
);
