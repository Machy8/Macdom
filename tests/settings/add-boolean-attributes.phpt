<?php

use Tester\Assert;

require '../bootstrap.php';

$tested = 'input $text beer steak muhehe';
$result = '<input type="text" beer steak muhehe>';

$o->setup->addBooleanAttributes = "beer steak muhehe";
$o->setup->compressCode = TRUE;

Assert::same($result, $o->compileContent($tested));
