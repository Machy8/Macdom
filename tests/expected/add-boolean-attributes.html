<input type="text" beer steak muhehe>